### PR TITLE
#89 refactor consultant page (error handling & template part)

### PR DESCRIPTION
**Pull request by knut-hufthammer**
This PR includes the following changes:

1.  Handle case when no consultant is found (404 redirect)
2. Refactor (fetch logic in template & display logic in template part)

### CODE REVIEW

File: template-folk.php [hunk 1]
```diff
@@ -71,67 +71,18 @@ function format_query_vars()
 	);
 }
 
-if (!$cv || $cv == 'null') {
-	echo 'No result. Ensure you have the API set up correctly following the guide here - https://wpgetapi.com/docs/quick-start-guide/';
-} else {
-	$cvObject = json_decode($cv, true);
+if ($cv) {
```
Comment (mahdiislam) [inline template-folk.php:74]: What happens if `wpgetapi` fails altogether? `$cv` will be `null`, should we show some error in that case?

File: template-folk.php [hunk 2]
```diff
@@ -71,67 +71,18 @@ function format_query_vars()
...
-
-<?php
-
-get_footer();
\ No newline at end of file
+	set_query_var('cv', $cvObject);
+	get_template_part('template-parts/content-consultant');
+	get_footer();
+}
+?>
```
Comment (mahdiislam) [inline template-folk.php:88]: Should we have an end tag `?>` here? Or remove it (https://wordpress.stackexchange.com/questions/210765/to-close-or-not-to-close-php)?

**Review by mahdiislam — COMMENTED** [review]
Please have a look.

File: template-folk.php (hunk 1 above)
Comment (knut-hufthammer) [reply template-folk.php:74]: Good spot! Will Fix!

File: template-folk.php (hunk 2 above)
Comment (knut-hufthammer) [reply template-folk.php:88]: Will fix 🙏

**Review by mahdiislam — APPROVED** [review]
Looks good!

**Pull request merged**